src/components/Alert/index.tsx

```diff
@@ -83,7 +83,8 @@ const StyledIcon = styled(Icon)`
   margin-right: ${({ theme }) => theme.space[2]};
 `
 
-const StyledBox = styled(Box)<{ color: string }>`
+const AlertContainer = styled.div`
```

Comment:
lot of breaking change on our console project. Be carefull to update all brefore upgrade

Reply:
No breaking changes, this is an inner component :)